scripts/ci/cut_ironclaw_release.py

```
@@ -0,0 +1,150 @@
+        raise ReleaseTagError(
+            f"approved commit declares version {manifest_version}, "
+            f"not requested version {requested_version}"
+        )
+
+    tag = f"ironclaw-v{requested_version}"
+    existing_target = get_tag_target(tag)
+    if existing_target is not None:
+        if existing_target != requested_sha:
+            raise ReleaseTagError(
+                f"{tag} already points to {existing_target}, not {requested_sha}"
+            )
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)